doc/api/util.md

```diff
@@ -1989,6 +1989,35 @@ const channel = new MessageChannel();
 channel.port2.postMessage(signal, [signal]);
 ```
 
+## `util.aborted(signal, resource)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `signal` {AbortSignal}
+* `resource` {any} Any non-null entity
+* Returns: {Promise}
+
+Listens to abort event on the provided `signal` and returns a promise which is
+resolved if an abort event is triggered. The function cleans up any
+event listeners depending on when the provided `resource` is garbage collected.
+
+```js
+const { aborted } = require('util');
+
+const ac = new AbortController();
+const dependent = {};
```

Comment:
I don't like this example since you're supposed to pass a resource in - the issue is all our resources already take signals :D so maybe a mock call like `const dependent = obtainSomethingAbortable()` and calling `dependent.abort()` in the listener?

Reply:
Ah ok xD, updating